support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutFormGifting.tsx

```diff
@@ -130,34 +95,37 @@ function mapStateToProps(state: WithDeliveryCheckoutState) {
 		isTestUser: state.page.checkout.isTestUser,
 		country: state.common.internationalisation.countryId,
 		csrf: state.page.csrf,
-		currencyId: currencyFromCountryCode(deliveryAddress.fields.country),
+		currencyId:
```

Comment:
`currencyId` cannot be `null`, however `currencyFromCountryCode` can return null if the country code currency lookup can't find a matching currency, in this case I've opted to fallback to the `defaultCurrency` for the region.